### ISSUE TITLE
Experimental results are not as good as expected

### ISSUE THREAD

**Issue by RonaldJEN** [vub-ai-lab/bdpi#3]
Describe the Question
Hello,
I have two questions after reading your paper.First, you used  two Q-functions QA & QB .Every training iteration, QA and QB are swapped,Is this like Delayed network that makes network updates more stable?
Second, I try an experiment in LunarLander-v2 based on your code, and it was not as stable as I expected, average reward not state of the art. The following were the details:
* Parse parameters(Benchmark on LunarLander-v2)
<img width="1045" alt="parse" src="https://user-images.githubusercontent.com/36885328/71775090-b7f38f80-2fb5-11ea-88a2-2e3ee022110a.png">
* experiment
<img width="1045" alt="env_LunarLander" src="https://user-images.githubusercontent.com/36885328/71782745-8cee5780-3018-11ea-8c4b-b3bc54431bca.png">

* System Info
Describe the characteristic of your environment:
Describe how the library was installed (pip, docker, source, ...)
Python version : 3.5.6
Looking forward to your kindly reply , thank you.

**Comment by steckdenis** [comment]
Hello,

Do you have a link to a paper on "Delayed network" so that I can use the exact definition you have seen in my reply? I would say that having two neural networks, Qa and Qb, and swapping them, indeed makes the updates more stable. You can see a discussion of this in https://arxiv.org/abs/1802.09477 .

Regarding your hyper-parameters, I see a few ones that are not as described in the paper (they have weird names, I must confess):

- aepochs and cepochs must be set to 20, instead of 1. This will increase the sample-efficiency about 20-fold.
- q-loops should be set to 4, this increases stability and sample-efficiency
- you may try to lower "hidden" to 64 instead of 128. In the paper, I use 128, but I have since discovered that BDPI works a bit better with smaller neural networks. It seems to still be sensitive to overfitting.

Best regards,
Denis

**Comment by RonaldJEN** [comment]
hi @steckdenis   , thank for your detail explanation. "Delayed network" ,  I mean to describe the relationship between Qa and Qb , that like target network and prediction network. Based on your answer,
it also matched what I think as you. 
Regarding hyper-parameters:  I will try again the experiment based on your opinion.

Thank for your kindly reply.

Sincerely ,
RonaldJEN
